app/src/main/java/com/example/oddhours/data/repository/JobRepository.kt

```diff
@@ -91,14 +88,12 @@ class JobRepository() {
          * Looping through each jobId and getting shifts and adding to the List of ShiftsListModel
          */
         for(i in jobIDList){
```

Comment:
also maybe consider renaming the counter from `i`?

Reply:
yup, let me do that